src/docs/sphinx/quickstart.rst

```diff
@@ -243,7 +243,7 @@ install cmake, MPICH, openblas, OpenGL, and the various developer tools using th
 
    $ sudo apt-get update
    $ sudo apt-get upgrade
-   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev cppcheck doxygen libreadline-dev python3-sphinx python3-pip clang-format-10 m4
+   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev cppcheck doxygen libreadline-dev python3-sphinx python3-pip clang-format-10 m4 elfutils
```

Comment:
this part of quickstart is out of date. should the addition of `elfutils` be added to the dockerfiles in `scripts/docker`?